src/helpers/resolveJsonPath.ts

```diff
@@ -0,0 +1,8 @@
+export const resolvePath = (object: any, path: string) =>
```

Comment:
@abbasali1994, no need for this helper found in `react-hook-form/ErrorMessage` you could use [`get`](https://github.com/react-hook-form/error-message/blob/master/src/ErrorMessage.tsx#L21) util

Reply:
DOne